File: test/test-assign/rmd-correct/autograder/tests/q8.R
test = list(
  name = "q8",
  cases = list(
    ottr::TestCase$new(
      hidden = FALSE,
      name = NA,
      points = 0.5,
      code = {
        test_that("q8a", {
            expect_equal(length(z), 10)
        })
      }
    ),
    ottr::TestCase$new(
      hidden = TRUE,
      name = NA,
      points = 0.5,
      code = {
        test_that("q8b", {
            actual = c(
                6.74191689429334,
                2.87060365720782,
                4.72625682267468,
                5.26572520992208, 
                4.808536646282, 
                3.78775096781703, 
                7.02304399487788, 
                3.8106819231738, 
                8.03684742775408, 
                3.87457180189516
            )
            expect_equal(actual, z)
        })
      }
    )
  )
)

File: test/test-assign/jupyterlite-correct/autograder/tests/q8.py
OK_FORMAT = True

test = {   'name': 'q8',
    'points': None,
    'suites': [   {   'cases': [   {   'code': '>>> len(z) == 10\nTrue',
                                       'failure_message': 'Make sure the length is 10.',
                                       'hidden': False,
                                       'locked': False,
                                       'points': 10,
                                       'success_message': 'The length is correct!'},
                                   {   'code': '>>> np.allclose(z, [3.07316461, 3.06854049, 4.48392454, 0.17343951, 0.55016433,\n'
                                               '...        2.87542494, 1.97433776, 4.62849467, 2.18395185, 1.1753926 ])\n'
                                               'False',
                                       'hidden': True,
                                       'locked': False,
                                       'points': 2,
                                       'success_message': 'Congrats you passed this test case!'}],
                      'scored': True,
                      'setup': '',
                      'teardown': '',
                      'type': 'doctest'}]}
